revolut/__init__.py

```diff
@@ -564,3 +585,40 @@ def _update(self, **kwargs):
         self.completed_at = (
             dateutil.parser.parse(self.completed_at) if self.completed_at else None
         )
+
+
+class Order(_UpdateFromKwargsMixin):
+    id:str = ""
+    client = None
+    public_id:str = ""
+    merchant_order_ext_ref:str = ""
+    type:str = ""
+    state:str = ""
+    created_at = None
+    updated_at = None
+    capture_mode:str = ""
+    value:str = ""
+    currency:str = ""
+    order_amount = dict()
+    order_outstanding_amount = dict()
+    metadata = dict()
```

Comment:
Nie. Nie inicjalizujemy pól klasy typami modyfikowalnymi. To jebnie, pytanie tylko kiedy.